src/main/kotlin/com/autonomousapps/tasks/FailOrWarnTask.kt

```diff
@@ -37,13 +37,17 @@ abstract class FailOrWarnTask : DefaultTask() {
   @get:Input
   abstract val failOnIncorrectConfiguration: Property<Behavior>
 
+  @get:Input
+  abstract val failOnUnusedProcs: Property<Behavior>
+
   @TaskAction
   fun action() {
     val adviceReports = advice.get().asFile.readText().fromJsonSet<BuildHealth>()
 
     val anyUnusedDependencies = adviceReports.any { (_, advices, _) -> advices.any { it.isRemove() } }
     val anyUsedTransitives = adviceReports.any { (_, advices, _) -> advices.any { it.isAdd() } }
     val anyIncorrectConfigurations = adviceReports.any { (_, advices, _) -> advices.any { it.isChange() } }
+    val anyUnusedProcs = adviceReports.any { (_, advices, _) -> advices.any { it.isChange() } }
```

Comment:
do you mean to use `it.isChange()` at the end?

Reply:
No, it's a merge issue.
Thx

Reply:
I don't understand here how you wanna do without introducing a `isProcessor`.

Reply:
`isProcessor()` seems ok. It's the new factories for creating `Advice` instances I'm opposed to.